components/MobileNavigation.js

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from 'react'
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup'
+
+import LevitatingLink from './LevitatingLink'
+import Hamburger from '../resources/hamburger.svg'
+
+export default class MobileNavigation extends Component {
+  state = {
+    showMobileNav: false,
+    expandMobileNav: false
+  }
+
+  componentDidMount () {
+    window.addEventListener('scroll', this.handleScroll)
+  }
+
+  componentWillUnmount () {
+    window.removeEventListener('scroll', this.handleScroll)
+  }
+
+  handleScroll = (e) => {
+    window.scrollY > 700 ? this.showMobileNav() : this.hideMobileNav()
+  }
+
+  showMobileNav = () => {
+    this.setState({
```

Comment:
This function is called every `'scroll'` event, which means we make a lot of redundant `setState` calls. Before calling `setState`, please check if the state needs to be changed first.